meshpy/utility.py

```diff
@@ -143,17 +143,27 @@ def get_min_max_coordinates(nodes):
     return min_max
 
 
-def clean_simulation_directory(sim_dir):
+def clean_simulation_directory(sim_dir, *, ask_before_clean=False):
     """
-    If the simulation directory exists, the user is asked if the contents
-    should be removed. If it does not exist, it is created.
+    Clear the simulation directory. If it does not exist, it is created.
+    Optionally the user can be asked before a deletion of files.
+    Args
+    ----
+    sim_dir:
+        Path to a directory
+    ask_before_clean: bool
+        Flag which indicates whether the user must confirm removal of files and directories
     """
 
     # Check if simulation directory exists.
     if os.path.exists(sim_dir):
-        print(f'Path "{sim_dir}" already exists')
```

Comment:
Shouldn’t this be the other way? If ask_before_clean is True we want to show the message and ask the user. Now it is the other way around.

Reply:
yes, you are right, i have inverted the flag.